package.json

```diff
@@ -3,16 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.19.2",
```

Comment:
Need to take this out since I'm not using it anymore.

Reply:
Just using regular fetch() API?

Reply:
Yeap, I brought axios first thing when I realised I'd have to handle a lot of promise based api calls. Then I found a good tutorial that is using a middleware with redux. It's called react-thunk I think. Here to be more explicit. https://www.taniarascia.com/redux-react-guide/

I also need to remove that dependency since I'm not using it.